netutils/dhcpc/dhcpc.c

```diff
@@ -145,7 +147,7 @@ struct dhcpc_state_s
  * Private Data
  ****************************************************************************/
 
-static const uint8_t xid[4] =
```

Comment:
how about move the global array to corresponding function to save the rodata?

Reply:
Sure